docs/usecases/datastorage_for_institutions.rst

```diff
@@ -157,162 +423,232 @@ about where data are stored, as they can access it just as easily as before.
+
+   - unless it is explicitly given via ``--ria-remote-name``, the sibling to the
+     ria-remote special remote will have the same sibling name suffixed with ``-ria``.
+   - Special remote capabilities of a RIA store can be disabled at the time of
+     RIA store creation by using the option ``--no-ria-remote``
+   - :term:`Permissions` in the RIA store can be configured via ``--shared``. It
+     will default to multi-user access, but can take any specification of
+     the :command:`git init --shared` argument (find out more in the
+     `documentation <https://git-scm.com/docs/git-init>`__ of the command).
+
+
+RIA stores can be used under different types of file transfer protocols.
+Depending on the file transfer protocol, the looks of the URL can differ:
+
+- :term:`SSH`: ``ria+ssh://[user@]hostname:/absolute/path/to/ria-store``
+- Local file system: ``ria+file:///absolute/path/to/ria-store``
+- :term:`http` (e.g., to a RIA store like `store.datalad.org <http://store.datalad.org/>`_):
+  ``ria+http://store.datalad.org:/absolute/path/to/ria-store``
+
+Note that it is required to specify an :term:`absolute path` in the URL. Here is
+how one could create two siblings, ``server_juseless`` and ``server_juseless-ria``,
+to a RIA store (which can, but does not need to exist yet) on an :term:`SSH server`
+from within an existing dataset:
+
+.. code-block:: bash
+
+   $ datalad create-sibling-ria -s server_juseless \
+     ria+ssh://user@juseless.inm7.de:/home/user/scratch/myriastore
+   [INFO   ] create siblings 'server_juseless' and 'server_juseless-ria' ...
+   [INFO   ] Fetching updates for <Dataset path=/tmp/my_dataset>
+   [INFO   ] Configure additional publication dependency on "server_juseless-ria"
+   create-sibling-ria(ok): /tmp/my_dataset (dataset)
+
+   $ datalad siblings
+    .: here(+) [git]
+    .: server_juseless(-) [ (git)]
+    .: server_juseless-ria(+) [ria]
+
+Once the siblings to the RIA store are created, a
+:command:`datalad publish --to <sibling> --transfer-data all` publishes the
+dataset to the RIA store. With git-annex special remote capabilities enabled as
+in the example above, annexed contents will be published automatically.
+
+To clone a dataset from the RIA store, the RIA URL needs to be passed to the
+:command:`datalad clone` command, following a similar scheme as outlined above:
+
+- A ``ria+`` identifier for a RIA URL, followed by a protocol specification and
+  a path to the RIA store (identical to the URL before).
+- A ``#`` sign,
+- The :term:`dataset ID`,
+- (Optionally) a ``@`` followed by a version identifier such as a tag or a branch
+  name.
+
+Here is how to clone a dataset with the ID ``1d368e0a-439e-11ea-b341-d0c637c523bc``
+in the version identified by the tag ``ready4analysis`` from a RIA store on an
+SSH server:
+
+.. code-block:: bash
+
+   $ datalad clone \
+     ria+ssh://user@juseless.inm7.de/home/user/scratch/myriastore#1d368e0a-439e-11ea-b341-d0c637c523bc@ready4analysis \
+     mydataset
```

Comment:
I start to think that we should use store.datalad.org as the subject of all examples.